### ISSUE TITLE
Bookmarked anime won't show up in bookmark tab

### ISSUE THREAD

**Issue by wadeceol32** [Anistream/Anistream-Mobile#91]
After the most recent update i can only see a few of the anime i had bookmarked and when i went to an anime i know I had booked it showed that it was bookmarked but not showing up in the tab. Along with this issues I when I go into the dubbed tab it just has a gray screen with the search bar. Btw I thought you weren't supposed to have ads man,what happened to you. I can't watch durarara peacefully anymore because I have ads popping up.

**Comment by ShadoWarrior04** [comment]
And i guess the bookmarked anime not showing is the only fault in this app as of for now...

**Comment by Silver-Wulf** [comment]
Do newly added bookmarks show up?

**Comment by ShadoWarrior04** [comment]
Nope... Only 2 bookmarked anime was shown to me... When I tried to bookmark another one it didnt show any.

**Comment by wadeceol32** [comment]
> Do newly added bookmarks show up?

The ones I had only show up after I remove a bookmark that I can see,then it just gets replaced
